recipe/meta.yaml

```diff
@@ -12,23 +12,22 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 1
   skip: True  # [win]
 
 requirements:
   build:
-    - toolchain
     - pkg-config
     - protobuf
     - libprotobuf  # [linux]
-    - ncurses 5.9*
+    - ncurses 6.*
```

Comment:
We are not quite ready to make this switch yet. Still using [`ncurses` 5.9]( https://github.com/conda-forge/conda-forge.github.io/blob/06e20200a735ca2a22848caa6f6ad73c0f8f7419/scripts/pin_the_slow_way.py#L76 ) for now.

Reply:
Doesn't it give rise to problems for those who upgraded to anaconda 5.0?

Reply:
So the only issue I know of was related to `libedit`, which we have provided a workaround for. Admittedly we are going to want to `ncurses` upgraded across the stack, but that only works if we actually start from the bottom of the stack and work up. So `readline` will need to be fixed first. Otherwise one will have an issue trying to install `mosh` with `python` for example.